app/views/static/_about_text.html.erb

```diff
@@ -5,7 +5,7 @@
   The mission of <%=t('hyrax.product_name') %> is to preserve the permanent intellectual output of UC, to advance discovery and innovation, to foster scholarship and learning through the transformation of data into knowledge, to collect a corpus of works that can be used for teaching and to inspire derivative works, and to enhance discoverability and access to these resources.
 </p>
 <p>
-  <%=t('hyrax.product_name') %> is an open source, agile development project supported in partnership by the <%=t('hyrax.institution_name') %> Libraries and UC Information Technologies. Our source code repository is found on github.com in two places: <a href="https://github.com/uclibs/scholar_uc" target="_blank">our application code</a> and a <a href="https://github.com/projecthydra/hyrax" target="_blank">Ruby on Rails engine</a>.  Our application is built on top of <a href="http://projecthydra.org" target="_blank">Project Hydra</a>, a repository framework used by institutions worldwide to provide access to their digital content, and an open source community to facilitate shared software development.
+  <%=t('hyrax.product_name') %> is an open source, agile development project supported in partnership by the <%=t('hyrax.institution_name') %> Libraries and UC Information Technologies. Our source code repository is found on github.com in two places: <a href="https://github.com/uclibs/scholar_uc" target="_blank">our application code</a> and a <a href="https://github.com/projecthydra/hyrax" target="_blank">Ruby on Rails engine</a>.  Our application is built on top of <a href="http://projecthydra.org" target="_blank">Project Samvera</a>, a repository framework used by institutions worldwide to provide access to their digital content, and an open source community to facilitate shared software development.
```

Comment:
@lawhorkl Change `http://projecthydra.org` to `https://samvera.org` and change `https://github.com/projecthydra/hyrax` to `https://github.com/samvera/hyrax`